spec/services/doc_to_pdf_converter_spec.rb

```diff
@@ -0,0 +1,32 @@
+require "rails_helper"
+
+RSpec.describe DocToPdfConverter do
+  describe "#call" do
+    it "raises error if libreoffice not installed" do
+      source_file = build(:source_file)
+      allow(Kernel).to receive(:system).with("command -v soffice").and_return false
+
+      expect { DocToPdfConverter.call(source_file) }.to raise_exception DocToPdfConverter::DependencyNotFoundError
+    end
+
+    it "raises error if conversion failed" do
+      source_file = build(:source_file)
+      FileUtils.remove_dir(DocToPdfConverter::TMP_DIRECTORY, true) if File.exist?(DocToPdfConverter::TMP_DIRECTORY)
```

Comment:
Wondering if we should be consistent between using `FileUtils` vs `Dir`?

Reply:
@littleforest `Dir.rm_dr` raises an error if the folder is not empty and there's no way to force, while `FileUtils.remove_dir` allows it.

Reply:
Oh nice, good to know.